content/en/tracing/api_catalog/_index.md

```diff
@@ -0,0 +1,136 @@
+
+## Exploring your endpoints
+
+The [API Catalog][1] view shows all endpoints in all environments in your Datadog organization. The performance data shown for each endpoint is scoped to the environments and time frame you select. You can browse and ask valuable questions by querying on different properties and metrics to get more precise results, using facets and tags for quick filtering.
+
+In the list table, **sort** by any of the columns by clicking a header. For example, click **P99** to see endpoints with the highest 99th percentile for latency.
+
+The table also shows **team ownership** for the API. This information is inherited from the associated service's definition in the [Service Catalog][2]. A service owner owns all of the endpoints that are connected to the service.
+
+To **filter the list** of endpoints or to search for a particular endpoint of interest, enter a query in the **Search** field. Search by service, path, or any other primary tag you wish. Or select a combination of facets on the left.
+
+To **scope the data** shown in the table, specify an environment, another primary tag (such as datacenter), and a time frame.
+
+{{< img src="tracing/api_catalog/api-catalog-scope.mp4" alt="Showing how changing scope settings changes the metrics shown in the API Catalog" video="true" >}}
+
+You can [define tags](#group-apis-to-express-a-feature-or-business-logic) to use as facets so you can easily find groups of endpoints you're most interested in.
+
+**Note**: The `API Type` facet is based on auto-generated tags and has values of `public` and `unknown` (which usually indicates a private endpoint).
+
+
+## Investigating endpoint details
+
+When you're exploring an endpoint, getting a higher resolution view of the API endpoints details is helpful.
+
+Click an endpoint in the list to open a details page that shows performance, governance, and metadata information for the endpoint, collected from various areas of Datadog into one place. Here you can edit the short name and description, and add custom tags. You can also launch deeper investigations of the various types of telemetry using links into other areas of Datadog. 
+
+For example, you can:
+- Add facets to endpoints by the owning service or the team they belong to. For example, you could add a `web-store` service facet.
+- Look up a specific endpoint by path (for example, `/checkout`) when it shows high error rate and high request count.
+- Navigate to related telemetry like traces and logs.
+- View the Error/Hits graph and correlated Response Code graph to identify issues.
+
+The performance graphs on the page are initially scoped to the same settings as on the API Catalog page, and you can change those settings in the details page to suit your investigation by using the time frame selector and other scope drop-downs. 
+
```

Comment:
add a mp4 showing this, the navigation to traces from Endpoint 500 errors is pretty strong to show